src/scss/foundations/colors.scss

```diff
@@ -0,0 +1,132 @@
+@use "../lib";
+
+$_color-action-base: #067f96;
+$_color-action-dark: #e5f1f4;
+$_color-action-light: #045767;
+
+$_color-alert-base: #ffdc50;
+$_color-alert-dark: #fff8dc;
+$_color-alert-light: #ffc010;
+
+$_color-alpha-base: rgba(#21283b, 0.70);
+
+$_color-brand-base: #8334c2;
+$_color-brand-dark: #f2eaf8;
+$_color-brand-light: #5b2488;
+
+$_color-error-base: #df1654;
+$_color-error-dark: #fbe7ed;
+$_color-error-light: #b21143;
+
+$_color-neutral-25: #595d6a;
+$_color-neutral-50: #90939d;
+$_color-neutral-75: #c7c9cd;
+$_color-neutral-85: #dddee1;
+$_color-neutral-95: #f3f4f4;
+$_color-neutral-base: #21283b;
+$_color-neutral-light: #ffffff;
+
+$_color-shadow-action: rgba(#067f96, 0.40);
+$_color-shadow-base: #000000;
+$_color-shadow-brand: rgba(#8334c2, 0.40);
+$_color-shadow-dark: rgba(#595d6a, 0.40);
+
+$_color-success-base: #1d8147;
+$_color-success-dark: #e8f2ec;
+$_color-success-light: #145a31;
+
+// The color palette of Pyxis
+//
+// @type Map
+//
+// @access private
+
+$_colors: (
+  action: (
+    base: $_color-action-base,
+    dark: $_color-action-dark,
+    light: $_color-action-light,
+  ),
+  alert: (
```

Comment:
Qui idealmente mi aspetterei che tutti i colori abbiano lo stesso numero di `sfumature`. Alcuni però, tipo `neutral` sono un po' tanti. Cerchiamo di trovare un sistema "predittivo" per l'utilizzatore finale?

Reply:
Eliminando le `shadow` rimangono i neutral che sono effettivamente gli unici diversi, in quanto abbiamo molte più scale sui grigi (cosa che non serve su altri colori).

Al momento i colori neutral partono dal base `$_colorNeutralBase` a cui si aggiunge una percentuale di bianco:
```
$_colorNeutralBase + 25% di bianco = $_colorNeutral25
$_colorNeutralBase + 50% di bianco = $_colorNeutral50
...
```
questo metodo è usato in molti design system ([comet](https://comet.discoveryeducation.com/visual-language/color.html), [audi](https://github.com/audi/audi-ui/blob/master/src/_variables.scss) ed è in realtà abbastanza esplicativo. Avevo pensato di avvicinarli alle altre sfumature usando nomenclature come "ligher e darker", ma non bastano e non sono abbastanza flessibili se in futuro decidiamo di aggiungere una nuova sfumatura di grigi.

Ho anche provato a usare le funzioni mix (o altre) di sass per fare in modo da avere:
```
mix($_colorNeutralLight, $_colorNeutralBase, 25%)
```
ma non ritornano gli esatti HEX definiti sul design.

Al momento non mi viene in mente nulla per rendere un po' più predittiva questa cosa, però accetto consigli da tutti :P

Reply:
I `neutrals` sono tutti i `grigi` che usiamo all'interno del DS.
Possono essere usati come `background-color`, `border-color` o `color`. (Come mai per te non sono "neutrali"?)

In `pyxis-npm` erano divisi tra:
```
text
shape
background
backgroundAlt
```
ma spesso venivano usati in mix o male ed alcune varianti erano identiche (tipo `shape-light` e `background-dark`), così abbiamo deciso di racchiuderli sotto lo stesso tetto, avendo tutti come base: `$_colorNeutralBase`.